src/components/LinkButton/utils.ts

```diff
@@ -0,0 +1,79 @@
+import { useTheme } from '@mui/material';
+import { Colors, TypographyFontClass, TypographyFontSize } from '../../mui-theme';
+
+const getLinkButtonStyle = (
+  type: 'mint' | 'white' | 'grey' | 'red'
+): { BgColor: keyof Colors; hoverBgColor: keyof Colors } => {
+  let BgColor: keyof Colors = 'mint75';
+  let hoverBgColor: keyof Colors = 'mint90';
+
+  switch (type) {
+    case 'mint':
+      BgColor = 'mint75';
+      hoverBgColor = 'mint90';
+      break;
+
+    case 'white':
+      BgColor = 'lightWhite';
+      hoverBgColor = 'lightL3';
+      break;
+
+    case 'grey':
+      BgColor = 'dark75';
+      hoverBgColor = 'dark90';
+      break;
+
+    case 'red':
+      BgColor = 'error75';
+      hoverBgColor = 'error90';
+      break;
+
+    default:
+  }
+  return { BgColor, hoverBgColor };
+};
+
+const getLinkButtonFontStyle = (
+  size: 'lg' | 'md' | 'sm'
+): { paddingTop: string; paddingBottom: string; fontClass: TypographyFontClass; fontSize: TypographyFontSize } => {
+  const theme = useTheme();
```

Comment:
as useTheme is a react hook, your function name should follow the pattern `use<somethingHere>` to indicate that it's a hook. from the [react docs](https://legacy.reactjs.org/docs/hooks-custom.html#extracting-a-custom-hook):
![Screen Shot 2023-07-17 at 4 49 31 pm](https://github.com/beforeyoubid/ui-lib/assets/67300978/442550a6-b3b7-4d33-a984-5520d6e59bd0)